sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/keyvault_secret.hpp

```diff
@@ -9,11 +9,15 @@
 #pragma once
 #include <azure/keyvault/secrets/keyvault_secret_properties.hpp>
 
-#include <unordered_map>
-
 namespace Azure { namespace Security { namespace KeyVault { namespace Secrets {
   struct KeyVaultSecret final
   {
+    /**
+     * @brief The name of the secret.
+     *
+     */
+    std::string Name;
```

Comment:
I should mention that we also declare Version, Id, and VaultUri (well, VaultUrl for you) on this class, but they are just hoisted from SecretProperties. Might not make sense in this case, though.